.github/guides/HARDDELETES.md

```diff
@@ -263,3 +276,14 @@ First, do a quick check.
 Are you doing anything to the object in `Initialize()` that you don't undo in `Destroy()`? I don't mean like, setting its name, but are you adding it to any lists, stuff like that
 
 If this fails, you're just gonna have to read over this doc. You can skip the theory if you'd like, but it's all pretty important for having an understanding of this problem
+
+## Misc facts
+
+> i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code
```

Comment:
i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code